components/editor/utils/RepoSearch.js

```diff
@@ -98,6 +114,28 @@ export const filterRepos = gql`
   }
 `
 
+const RepoItem = withT(({ t, repo }) => {
+  const title =
+    repo.latestCommit.document.meta.title ||
+    repo.id.replace([GITHUB_ORG, REPO_PREFIX || ''].join('/'), '')
+  const publicationDate =
+    repo?.latestPublications[0]?.document.meta.publishDate ||
```

Comment:
```suggestion
    repo.latestPublications[0]?.document.meta.publishDate ||
```

(Assume `repo` is an object.)